bedrock/base/templates/macros-protocol.html

```diff
@@ -291,7 +291,7 @@
 
 
 {# Picto: https://protocol.mozilla.org/patterns/molecules/picto.html #}
-{% macro picto(title=None, class=None, heading_level=3, base_el='div', body=False, image_url=None, image_width=64, include_highres_image=False, l10n_image=False, lazy_loading=False) -%}
```

Comment:
there's a "startsWith" check on the image_url that will error if default None is used, an empty string default allows this check